test/unit/routes/RoutesWrapper.spec.ts

```diff
@@ -4,18 +4,18 @@ import Route, { RouteWrapperDeclared } from 'src/types/Route';
 
 describe('Routes :: RoutesWrapper', () => {
     describe('When declare routes method is called', () => {
-        let routesDeclared: RouteWrapperDeclared[][];
+        let routesDeclared: RouteWrapperDeclared[];
 
         it('should return an array with 70 arrays inside - dungeons&dragons5e', () => {
             routesDeclared = RoutesWrapper.declareRoutes()['dungeons&dragons5e'];
             expect(routesDeclared).toBeInstanceOf(Array);
             expect(routesDeclared.length).toBe(70);
         });
 
-        it('should return an array with 8 arrays inside - user', () => {
+        it('should return an array with 9 routes inside - user', () => {
```

Comment:
São 18 rotas, "Should return an array with 18 routes"

Reply:
Vou alterar e subir